src/main/java/com/auctionapp/api/model/entities/User.java

```diff
@@ -0,0 +1,46 @@
+package com.auctionapp.api.model.entities;
+
+import java.sql.Timestamp;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Table;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
+@NoArgsConstructor
+public class User extends EntityWithUUID {
+
+	@Column(name = "first_name", nullable = true)
```

Comment:
Not sure you need this annotation - if the naming between tables and entities is consistent and appropriate, this should be mapped on its own. Also, why is the name nullable?

As a note, if you have fields in a class which should not ever be null, you should set a precondition in the constructor to make sure they aren't using `Objects.requireNonNull`.